### DISCUSSION TITLE
Question about Reporting status

### DISCUSSION THREAD

**Discussion by statisticamd** [open-sdg/open-sdg#2072] [Q&A] [answered]
Hello!

We have a problem with the display of the reporting status on the goals page.
We added 2 additional statuses - Divided and Not relevant. On the Reporting status page https://statisticamd.github.io/sdg-site-moldova/reporting-status the new statuses are displayed, but on the goals page https://statisticamd.github.io/sdg-site-moldova/1/ they are displayed randomly as Available and Not available status. For ex. for goal 1, indicators 1.3.1, 1.4.2, 1.5.1, 1.a.2 - have status Divided and 1.5.3 and 1.a.1 - Not relevant. We noticed that status in progress is not shown on the goal page - just a small grey box appears. Couldn't find how to add the text.

The statuses are translated here: https://github.com/statisticamd/sdg-translations/blob/2.2.0-dev/translations/en/status.yml  

Our repositories:
https://github.com/statisticamd/sdg-data-moldova
https://github.com/statisticamd/sdg-site-moldova
https://github.com/statisticamd/sdg-translations

And another question about Reporting status - is it possible to change the calculation of the percentage on this page https://statisticamd.github.io/sdg-site-moldova/reporting-status/? We would like to calculate the percentage only for indicators with 2 statuses - Available and Not available.

Thank you!

**Comment by sarah-beasley** [comment]
Hi, Thanks for your query and providing the relevant links.

I've been looking into your platform and reporting status page to better understand what is going on. I've noticed that there are 431 indicators on your reporting status page which is much higher than the 247 prescribed UN SDG indicators. Having looked into the goal pages, I think it's because additional indicators have been added when we would usually add these to the original indicator as series instead.
For example for 1.3.1 - your goal page says it's not available but actually there are a number of indicators underneath (1.3.1.a, 1.3.1.b etc.). 
![image](https://github.com/open-sdg/open-sdg/assets/106670682/9075049e-f763-421c-8bae-4c3014607922)

The structure of Open SDG is intended for users to put these within 1.3.1 as different series, which is what most other countries have done, just like the UK site [here for example.](https://sdgdata.gov.uk/1-3-1/), so this negates the need to add them as separate indicators.

I'm not sure if this is what was intended, but it would help make sure your reporting status is more accurate and consistent with the UN indicators and also other countries.

I will look into how to calculate the reporting status just based on available and not available, rather than the additional divided and not relevant categories. Can I please check just for my own understanding, what the divided term refers to?

I will also look into why the progress reporting options are not appearing on the site despite the indicator config forms having the required field.

**Reply by statisticamd** [reply]
Dear Sarah,

Thank you for your response!

Yes, indeed we have more indicators because we divided some global indicators into separate indicators that we call Global divided. Thus, we have data for the indicator 1.3.1 a, 1.3.1.b etc., but no data for 1.3.1.

Some indicators we marked as Not relevant (for ex. 1.5.3), because they were not relevant for our country and we haven't nationalized them, but instead we can have a national indicator related to the global one, for ex. 1.5.3.1. In our nationalized SDG list of indicators, we have a total of 337 indicators, and we would like to see on the Reporting status page specifically these indicators, it would be useful when we report the availability of data according to our nationalized list of indicators.

A solution would be to delete from the platform the indicators that were divided and Not relevant indicators (98 indicators), but we wanted to keep the whole list of global indicators on the platform.

Also, we were thinking if these indicators could remain on the platform, but they would not be clickable, not sure this is possible.

Thank you in advance for your help!

**Comment by brockfanning** [comment, marked as answer]
@statisticamd Regarding the first question, about why some are appearing as "Available" when they should not, etc, I would recommend that you upgrade to 2.2.0. I believe that may have been fixed in 2.2.0.

You also asked about the possibility of having the percentage calculation only include "Available" and "Not available". We don't have a way to do this, however, a solution might be the following:
1. For reporting status, have only two statuses: either "Available" or "Not available" -- and all indicators would have to be one of these two statuses.
2. For any other tags needed, use the ["Tags" indicator configuration feature](https://open-sdg.readthedocs.io/en/latest/indicator-configuration/#tags). For example, you could have "Divided" as a tag, instead of a reporting status.

You also asked about having indicators that remain on the platform, but are not clickable. I believe this is what the ["Placeholder" indicator configuration](https://open-sdg.readthedocs.io/en/latest/indicator-configuration/#placeholder) is for. Note that this also prevents the indicator from affecting the status percentages, as far as I understand.

**Reply by statisticamd** [reply]
Thank you very much Brock! 
We tried adding placeholder and it worked, it is non-clickable and it is not calculated in the Reporting status page, I think we will go with this solution.
Thank you!
